src/Dropdown/Dropdown.js

```diff
@@ -237,15 +239,13 @@ export class Dropdown extends Component {
       <div
         className={classNames('Dropdown', { 'is-open': isOpen }, className)}
         ref={this.dropdownRef}
-        role="listbox"
-        aria-expanded={isOpen}
-        aria-haspopup="listbox"
-        aria-owns={`${name}-options`}
-        aria-label={label}
       >
         <button
           aria-controls={`${name}-options`}
-          aria-label={this.selectedOption.label}
+          role="listbox"
+          aria-label={label}
+          aria-expanded={isOpen}
+          aria-activedescendant={`${name}-option-${selectedIndex}`}
```

Comment:
at the end, can a button have the active-descentrant prop?

Reply:
Yes. 
What we can't have is an element with **role**="button" and aria-activedescendant.The attribute aria-activedescendant is for elements with **role**="listbox", regardless of the element type (button, span, a, div)

Reply:
perfect,  thanks for the clarification